src/tsoa/routes.ts

```diff
@@ -0,0 +1,1573 @@
+/* tslint:disable */
+/* eslint-disable */
+// WARNING: This file was auto-generated with tsoa. Please do not modify it. Re-run tsoa to re-generate this file: https://github.com/lukeautry/tsoa
+import { Controller, ValidationService, FieldErrors, ValidateError, TsoaRoute, HttpStatusCodeLiteral, TsoaResponse } from 'tsoa';
+// WARNING: This file was auto-generated with tsoa. Please do not modify it. Re-run tsoa to re-generate this file: https://github.com/lukeautry/tsoa
+import { AuthRouter } from './../routes/AuthRouter';
```

Comment:
Might be able to put these into routes/index.ts ?

Reply:
This file is generated, you would need to copy/paste the content everytime your api changes

Reply:
Why wouldn't it be possible to generate imports from routes/index.ts?